kernel/tests/read.rs

```diff
@@ -393,7 +393,7 @@ fn read_with_scan_metadata(
                 read_result,
                 scan.physical_schema(),
                 scan.logical_schema(),
-                &scan_file.transform,
+                scan_file.transform.clone(),
```

Comment:
far from a site where i immediately see it's an Arc so might be useful to call out?
```suggestion
                scan_file.transform.clone(), // Arc clone
```

Reply:
It's also an integration test... do we care if it's cheap or expensive?